package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -456,11 +456,11 @@ class DumpReader(base.ReaderBase):
     "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
     conventions (see https://docs.lammps.org/dump.html for more details).
     If `lammps_coordinate_convention='auto'` (default),
-    one will be guessed. Guessing checks whether the coordinates fit each convention in the order "unscaled",
-    "scaled", "unwrapped", "scaled_unwrapped" and whichever set of coordinates
-    is detected first will be used. If coordinates are given in the scaled
-    coordinate convention (xs,ys,zs) or scaled unwrapped coordinate convention
-    (xsu,ysu,zsu) they will automatically be converted from their
+    one will be guessed. Guessing checks whether the coordinates fit each convention 
+    in the order "unscaled", "scaled", "unwrapped", "scaled_unwrapped" and whichever 
+    set of coordinates is detected first will be used. If coordinates are given in 
+    the scaled coordinate convention (xs,ys,zs) or scaled unwrapped coordinate 
+    convention (xsu,ysu,zsu) they will automatically be converted from their
```

Comment:
You will need to add documentation of the new option, preferably with an example. 

Also, you will need a `.. versionchanged entry`